src/lib.rs

```diff
@@ -474,17 +474,20 @@ abort:
 */
 
 .section .trap, "ax"
-.balign 0x100
 .global _vector_table
 .type _vector_table, @function
+
+.option push
+.balign 0x100
 .option norelax
+.option norvc
 
 _vector_table:
-    .option push
-    .option norvc
+    j _start_trap
```

Comment:
I pulled this out ahead of the `.rept` instead of increasing the count because 1) that felt like it captured the intent/spirit of the architecture a little better, and 2) I'm also concurrently digging into some performance questions on our project, and I suspect having a different `_start_exc` or similar here in the first slot will end up being useful for that.

Reply:
I like this. I also thought about making the exception handler overridable while not touching the interrupt handlers - this would be a first step